docs/ManualDeploymentGuideScalarDLAuditorOnAKS.md

```diff
@@ -1,125 +1,98 @@
-# Deploy Scalar DL Ledger and Scalar DL Auditor on AKS (Azure Kubernetes Service)
+# Deploy ScalarDL Ledger and ScalarDL Auditor on AKS (Azure Kubernetes Service)
 
-Scalar DL is scalable and practical Byzantine fault detection middleware for transactional database systems, which achieves correctness, scalability, and database agnosticism.  
+ScalarDL is scalable and practical Byzantine fault detection middleware for transactional database systems, which achieves correctness, scalability, and database agnosticism.
 
-Scalar DL is composed of [Ledger](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started.md), [Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md), and [Client SDK](https://github.com/scalar-labs/scalardl/tree/master/docs#client-sdks). Scalar DL Ledger manages application data in its own unique way using hash-chain and digital signature. Scalar DL Auditor is an optional component and manages a copy of Ledger data without depending on Ledger to identify the discrepancy between Ledger and Auditor data. The Client SDK is a set of user-facing programs to interact with Ledger and Auditor.  
+ScalarDL is composed of [Ledger](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started.md), [Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md), and [Client SDK](https://github.com/scalar-labs/scalardl/tree/master/docs#client-sdks). ScalarDL Ledger manages application data in its own unique way using hash-chain and digital signature. ScalarDL Auditor is an optional component and manages a copy of Ledger data without depending on Ledger to identify the discrepancy between Ledger and Auditor data. The Client SDK is a set of user-facing programs to interact with Ledger and Auditor.
 
-We can deploy **Scalar DL Ledger** and **Scalar DL Auditor** on any Kubernetes services. This document explains how to deploy **Scalar DL Ledger** and **Scalar DL Auditor** on AKS.  
+We can deploy **ScalarDL Ledger** and **ScalarDL Auditor** on any Kubernetes services. This document explains how to deploy **ScalarDL Ledger** and **ScalarDL Auditor** on AKS. 
 
 ## What we create
 
-In this guide, we create the following environment on your Azure account.  
+In this guide, we create one of the following three environments on your Azure account. To make byzantine fault detection work properly, we recommend deploying ScalarDL Ledger and ScalarDL Auditor on different administrative domains (separated environments).
```

Comment:
ScalarDL Auditor mode has three options for its deployment to separate administrative domains.

1. Use different cloud service accounts (most recommended way).
2. Use different private networks on one cloud service account (second recommended way).
3. Use different k8s namespaces on one k8s cluster (third recommended way).

Basically, the intensity of the separation weakens as one moves downward.